config/taxonomy/opa.structs.schema.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "opa.schema.json",
+    "title": "OPA input Taxonomy",
+    "description": "This is the OPA taxonomy.",
```

Comment:
Something a little more meaningful here would be good.  Ex: This taxonomy defines the structure sent to Open Policy Agent policy engine, which determines whether or how the data requested may be used by the requester.